documentation/config_file_auth.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: auth.conf"
+canonical: "/puppetserver/latest/config_file_auth.html"
+---
+
+[`trapperkeeper-authorization`]: https://github.com/puppetlabs/trapperkeeper-authorization
+[new `auth.conf`]: ./conf_file_auth.html
+[Puppet `auth.conf`]: /puppet/latest/reference/config_file_auth.html
+[deprecated]: ./deprecated_settings.html
+[`puppetserver.conf`]: ./conf_file_puppetserver.html
+
+The `auth.conf` file contains rules for authorizing access to Puppet Server's HTTP API endpoints. For a broader overview of Puppet Server configuration, see the [configuration documentation](./configuration.html).
+
+> **Deprecation Note:** As of version 2.2, Puppet Server can use [`trapperkeeper-authorization`][] for authentication, which is configured by rules and settings in `auth.conf` using a new HOCON configuration file format, and the new `conf.d` location for Puppet Server configuration files. The legacy [Puppet `auth.conf`][] whitelist-based authorization method and configuration file format are [deprecated][] and will be removed in a future version of Puppet Server.
```

Comment:
Would it be better to say "The legacy Puppet 'auth.conf' *and* whitelist-based authorization method..."?  It's basically two things that we're deprecating in favor of tk-authorization for Puppet Server:

- Puppet's auth.conf for requests made to the master service endpoints
- client-whitelist settings for requests made to the certificate_status(es) and puppet-admin endpoints